lib/fluent/plugin/out_mongo.rb

```diff
@@ -172,7 +172,6 @@ def client
     def collect_records(chunk)
       records = []
       chunk.msgpack_each {|time, record|
-        record[@time_key] = Time.at(time || record[@time_key]) if @include_time_key
```

Comment:
Hmm..., I see.
But I think this feature should achieve via `#inject_values_to_record` and `compat_parameters_convert(conf, :inject)`.